packages/angular_devkit/schematics/collection-schema.json

```diff
@@ -4,6 +4,11 @@
   "title": "Collection Schema for validating a 'collection.json'.",
   "type": "object",
   "properties": {
+    "extends": {
```

Comment:
How hard to add `type: [ "array", "string" ]` ?

Reply:
Added support for both.